src/pages/multipleProductValidity.tsx

```diff
@@ -1,25 +1,184 @@
+                                            </div>
+                                            <div className="govuk-radios__item">
+                                                <input
+                                                    className="govuk-radios__input"
+                                                    id={`calendar-day-row${index}`}
+                                                    name={`validity-row${index}`}
+                                                    type="radio"
+                                                    value="endOfCalendarDay"
+                                                />
+                                                <label
+                                                    className={`govuk-label govuk-radios__label validity-radio-button-margin ${
+                                                        errors.length > 0 ? 'validity-radio-button-error-margin' : ''
+                                                    }`}
+                                                    htmlFor={`calendar-day-row${index}`}
+                                                >
+                                                    <span className="visually-hidden-label">Calendar</span>
+                                                </label>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </fieldset>
+                            ))}
+                        </>
+                    </FormElementWrapper>
+                </div>
+                <input
+                    type="submit"
+                    value="Continue"
+                    id="continue-button"
+                    className="govuk-button govuk-button--start"
+                />
+            </form>
+        </main>
+    </Layout>
+);
+
+export const getServerSideProps = (ctx: NextPageContext): { props: MultipleProductValidityProps } => {
+    const cookies = parseCookies(ctx);
+    const operatorCookie = cookies[OPERATOR_COOKIE];
+    const numberOfProductsCookie = cookies[NUMBER_OF_PRODUCTS_COOKIE];
+    const multipleProductCookie = cookies[MULTIPLE_PRODUCT_COOKIE];
+
+    if (!operatorCookie || !numberOfProductsCookie || !multipleProductCookie) {
+        throw new Error('Necessary cookies not found to display the multiple product validity page');
+    }
+
+    const { operator } = JSON.parse(operatorCookie);
+    const numberOfProducts: string = JSON.parse(numberOfProductsCookie).numberOfProductsInput;
+    const multipleProducts: Product[] = JSON.parse(multipleProductCookie);
+
+    const errors: ErrorInfo[] = [];
+    if (multipleProducts.some(el => el.productValidity && el.productValidity.error !== '')) {
+        const errorHref = 'multiple-product-validity-radios-error';
+        const error: ErrorInfo = { errorMessage: 'Select one of the two validity options', id: errorHref };
```

Comment:
In the gov examples I've seen this summary error message matches the error shown on the input